caluma/caluma_workflow/jexl.py

```diff
@@ -12,6 +12,11 @@ def visit_Transform(self, transform):
         if transform.name == "groups" and not isinstance(transform.subject.value, list):
             yield f"{transform.subject.value} is not a valid list of groups."
 
+        if transform.name == "dynamic_group" and not isinstance(
+            transform.subject.value, str
+        ):
+            yield f"{transform.subject.value} is not a valid dynamic group name."
```

Comment:
As discussed: let's use the existing `groups` transform instead (check if it is dynamic group, otherwise treat as regular group).